docs/docs/internals/syntax.md

```diff
@@ -295,7 +295,7 @@ TypTypeParamClause::=  ‘[’ TypTypeParam {‘,’ TypTypeParam} ‘]’
 TypTypeParam      ::=  {Annotation} id [HkTypeParamClause] SubtypeBounds
 
 HkTypeParamClause ::=  ‘[’ HkTypeParam {‘,’ HkTypeParam} ‘]’
-HkTypeParam       ::=  {Annotation} [‘+’ | ‘-’] (Id[HkTypeParamClause] | ‘_’)
+HkTypeParam       ::=  {Annotation} [‘+’ | ‘-’] (id [HkTypeParamClause] | ‘_’)
```

Comment:
Looks like this line wasn't fully updated like in the spec:
```suggestion
HkTypeParam       ::=  {Annotation} (id [HkTypeParamClause] | ‘_’)
```

Reply:
I plan to go back and unify all the different type parameter kinds in the grammar. It's getting out of hand the way it is.